metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -85,6 +85,15 @@ spec:
 If you are using Red Hat OpenShift, you need to specify additional settings in
 the manifest file and enable the container to run as privileged.
 
+. Modify the `DaemonSet` container spec in the manifest file:
++
+[source,yaml]
+-----
+  securityContext:
+    runAsUser: 0
+    privileged: true
+-----
```

Comment:
Would it be good to place this inside of the DaemonSet commented out, that way it can be quickly commented out.

Reply:
actually, would it be possible to use the `oc patch` command to streamline adding this?

Reply:
@blakerouse we already have it https://github.com/elastic/beats/blob/4c84db1d7236ef52e82de795de31e96f0c7df2a0/deploy/kubernetes/metricbeat-kubernetes.yaml#L142 (and some for Fileveat).

@exekias it is possible with `oc -n kube-system patch ds metricbeat -p '{"spec": {"template": {"spec": {"containers": [{"name": "metricbeat", "securityContext": {"privileged": true, "runAsUser": 0}}]}}}}'` but it will restart the pods of the DS. Is this really wanted?

Reply:
yeah, by looking at the command and restart behavior, sounds like uncommenting may actually be cleaner